_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -57,6 +57,23 @@ default cmake.build_dir             {${workpath}/build}
 # minimal/initial value for the install rpath:
 default cmake.install_rpath         ${prefix}/lib
 
+# CMake provides several different generators corresponding to different utities
+# (and IDEs) used for building the sources. We support "Unix Makefiles" (the default)
+# and Ninja, a leaner-and-meaner alternative.
+#
+# In the Portfile, use
+#
+# cmake.generator Ninja
+# or
+# cmake.generator Unix Makefiles    # no quotes!
```

Comment:
@RJVB, is this now fixed with [your final commit](https://github.com/RJVB/macstrop/commit/793d814d31d3328355e7edf0f7bc116a6ca1a971) in `macstrop`?